examples/asr/hkust/mtl_transformer.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I would add results to the beginning of each config file, commented out by default of course. This way people know what to expect by running the config.

Reply:
Sure, I‘ll update current result w/o mpc